operations/service.go

```diff
@@ -100,7 +100,9 @@ func startSystemCronJobs(ctx context.Context, env evergreen.Environment) {
 
 	amboy.IntervalQueueOperation(ctx, env.RemoteQueue(), 15*time.Minute, time.Now(), opts, amboy.GroupQueueOperationFactory(
 		units.PopulateCatchupJobs(30),
-		units.PopulateHostAlertJobs(20)))
+		units.PopulateHostAlertJobs(20),
+		units.PopulatePruneRemoteQueueJobs(),
```

Comment:
this seems like a bit infrequent?

Reply:
the TTL is a week, so I don't think it needs to run very often